install/kubernetes/cilium/templates/cilium-operator/clusterrole.yaml

```diff
@@ -38,18 +38,29 @@ rules:
   - services/status
   verbs:
   - update
+- apiGroups:
+  - ""
+  resources:
+  # to check apiserver connectivity
+  - namespaces
+  - secrets
+  verbs:
+  - get
+  - list
+  - watch
 - apiGroups:
   - ""
   resources:
   # to perform the translation of a CNP that contains `ToGroup` to its endpoints
   - services
   - endpoints
-  # to check apiserver connectivity
-  - namespaces
   verbs:
   - get
   - list
   - watch
+  - create
+  - update
+  - delete
```

Comment:
Why do we need to perform these verbs? Please add a comment in the commit or in here.

Reply:
the operator will watch ingress objects, and then create child objects {service, endpoint, cec} accordingly, hence CRUD permissions are required for operator role. 

Commit message is updated accordingly, thanks :)